android-json-form-wizard/src/test/java/com/vijay/jsonwizard/widgets/TimePickerFactoryTest.java

```diff
@@ -17,75 +15,61 @@
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
-import org.mockito.ArgumentMatchers;
 import org.mockito.Mock;
 import org.mockito.Mockito;
 import org.mockito.MockitoAnnotations;
 import org.powermock.reflect.Whitebox;
+import org.robolectric.Robolectric;
 
 import java.util.List;
-import java.util.Locale;
 import java.util.Set;
 
 public class TimePickerFactoryTest extends BaseTest {
```

Comment:
extend `FactoryTest` instead, remove the `JsonFormActivity` declaration and assignment call super on `setUp`